web/pages/Groups.tsx

```
@@ -252,6 +255,25 @@ export const Groups = () => {
             required
             className={isNeo ? 'rounded-none' : ''}
           />
+          <div className="space-y-1.5">
+            <label className={`text-sm font-bold ${isNeo ? 'text-black uppercase' : (mode === 'dark' ? 'text-gray-300' : 'text-gray-700')}`}>
+              Currency
+            </label>
```

Biome (2.1.2)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)